docs/tutorial.md

```diff
@@ -52,23 +49,22 @@ rails new test-react-on-rails --webpack=react
 cd test-react-on-rails
 ```
 
-Note: you can do the following two commands in an existing Rails app or pre Rails 5.1.3:
+Note: if you are installing React On Rails in an existing app or an app that uses Rails pre 5.1.3, you will need to run these two commands as well:
 
 ```
 bundle exec rails webpacker:install
 bundle exec rails webpacker:install:react
 ```
 
-
-
 Add the **React On Rails** gem to your Gemfile:
 
 ```
-gem 'react_on_rails', '9.0.0'         # use latest gem version, prefer exact version 
+gem 'react_on_rails', '10.0.2'         # prefer exact version 
```

Comment:
`# prefer exact gem version to match npm version`

It seems like a good idea to explain the reason.

Reply:
Indeed it does! Thanks! I'll create another PR. I also spotted one more sentence I could improve.

Reply:
https://github.com/shakacode/react_on_rails/pull/981